modules/osdk-updating-v125-to-v128.adoc

```diff
@@ -0,0 +1,154 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/golang/osdk-golang-updating-projects.adoc
+// * operators/operator_sdk/ansible/osdk-ansible-updating-projects.adoc
+// * operators/operator_sdk/helm/osdk-helm-updating-projects.adoc
+// * operators/operator_sdk/helm/osdk-hybrid-helm-updating-projects.adoc
+// * operators/operator_sdk/java/osdk-java-updating-projects.adoc
+
+ifeval::["{context}" == "osdk-golang-updating-projects"]
+:golang:
+:type: Go
+endif::[]
+ifeval::["{context}" == "osdk-ansible-updating-projects"]
+:ansible:
+:type: Ansible
+endif::[]
+ifeval::["{context}" == "osdk-helm-updating-projects"]
+:helm:
+:type: Helm
+endif::[]
+ifeval::["{context}" == "osdk-hybrid-helm-updating-projects"]
+:hybrid:
+:type: Hybrid Helm
+endif::[]
+ifeval::["{context}" == "osdk-java-updating-projects"]
+:java:
+:type: Java
+endif::[]
+
+:_content-type: PROCEDURE
+[id="osdk-upgrading-projects_{context}"]
+= Updating {type}-based Operator projects for Operator SDK {osdk_ver}
+
+The following procedure updates an existing {type}-based Operator project for compatibility with {osdk_ver}.
+
+ifdef::golang[]
+[NOTE]
+====
+The `go/v3` plugin is being deprecated by Kubebuilder. As a result, the Operator SDK will also be migrating to the `go/v4` plugin in a future release.
+====
+endif::[]
+
+.Prerequisites
+
+* Operator SDK {osdk_ver} installed
+* An Operator project created or maintained with Operator SDK {osdk_ver_n1}
+
+.Procedure
+
+ifdef::helm,hybrid,java[]
+* Find the `ose-kube-rbac-proxy` pull spec in the following files, and update the image tag to `v4.13`:
+endif::[]
+ifdef::ansible,golang[]
+. Find the `ose-kube-rbac-proxy` pull spec in the following files, and update the image tag to `v4.13`:
+endif::[]
```

Comment:
Yes, the only difference is the `*` (for the assemblies where it will ultimately be a single-step procedure) and `.` (for the others with multiple steps).

Reply:
aah ok. I missed that. Thanks for clarifying.